fastchat/serve/cli.py

```diff
@@ -203,6 +204,14 @@ def main(args):
     else:
         raise ValueError(f"Invalid style for console: {args.style}")
     try:
+        dtype = None
+        if args.dtype == "float32":
+            dtype = torch.float32
+        elif args.dtype == "float16":
+            dtype = torch.float16
+        elif args.dtype == "bfloat16":
+            dtype = torch.bfloat16
```

Comment:
1. Move this to a common function `str_to_torch_dtype` in `fastchat/utils.py`
2. Raise exception in the last else block